code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -523,7 +523,7 @@
 	if(is_currently_ejecting)
 		return
 	var/list/mouse_control = params2list(params)
-	if(isAI(user) && !mouse_control["middle"])//AIs use MMB
+	if(isAI(user) == !mouse_control["middle"])//BASICALLY if a human uses MMB, or an AI doesn't, then do nothing.
```

Comment:
Also what is this

Reply:
I was told that MMB to use mech equipment should be exclusive to AIs. To that end, I need to check that either the user is not an AI *and* the click was not a MMB, or the user is an AI *and* the MMB was used. Thus, this will return early if only one of those two is true.

It's like a NOT XOR.

Reply:
I don't see a reason why humans should NOT be allowed to use mmb though, if they prefer that I don't see why we should take it away